template/.eslintrc.js

```diff
@@ -6,30 +6,68 @@ module.exports = {
     'plugin:react-native/all',
     'plugin:react/recommended',
   ],
-  plugins: ['@typescript-eslint', 'babel', 'react-hooks', 'react-native', 'react', 'prettier'],
+  plugins: [
+    '@typescript-eslint',
+    'babel',
+    'react-hooks',
+    'react-native',
+    'react',
+    'prettier',
+    'jest',
+    'eslint-comments',
+  ],
   env: {
     'react-native/react-native': true,
     es6: true,
     jest: true,
     node: true,
   },
+  globals: {
+    React: true,
+    JSX: true,
+  },
   rules: {
+    // Typescript Plugin
+    // The following rules are made available via `@typescript-eslint`.
```

Comment:
> The following rules are made available via `X`.

I don't think those comments (here and below for other plugins) are necessary. All those rules start with the plugin name anyway (`'@typescript-eslint/blah-blah'`).